### DISCUSSION TITLE
How can quarto books be customized to create an edited collection (rather than authored book)?

### DISCUSSION THREAD

**Discussion by IULibScholComm** [quarto-dev/quarto-cli#9358] [Q&A] [answered]
### Description

Is there a way to customize the Quarto Book format to make it more apt for the genre of an edited collection (a book with editors rather than authors wherein each chapter is written by different author(s))?

Specifically, 

- can the book metadata (that is displayed on the index page) be customized to replace "Author" with "Editor(s)"?
![metadata-author](https://github.com/quarto-dev/quarto-cli/assets/131798128/623bb6aa-edea-4db6-ab7e-a977a1e04b84)

- is the Quarto extension, [section-bibliographies](https://github.com/pandoc-ext/section-bibliographies), the best way to make separate bibliographies for each chapter (so that each chapter has its own bibliography), rather than the status quo of one bibliography for the whole book?

- if each chapter can have its own copyright license, can the License metadata field be displayed in each chapter, as is done for [Quarto articles](https://quarto.org/docs/authoring/front-matter.html)?

````qmd
---
title: "Introduction"
author: John Doe
abstract: >
    Essay about ...
license: "CC BY"
---

This is a book created from markdown and executable code.

See @knuth84 for additional discussion of literate programming.

````

 this metadata does not produce a license in a book chapter:
    
![license-book](https://github.com/quarto-dev/quarto-cli/assets/131798128/8328c993-87fd-4df4-bdef-5783ad4ebb65)

 but it does produce a license (reuse statement) in an article:

![license-article](https://github.com/quarto-dev/quarto-cli/assets/131798128/f327bd2e-1e4d-4457-b353-39c75096dcdc)

Thank you!

**Comment by mcanouil** [comment, marked as answer]
- see <https://quarto.org/docs/authoring/language.html>.
- there is no "best way" besides what works for you.
- you gonna need to use a LUA filter or use include shortcode with the copyright or whatever you want in it, and with your custom layout. In brief, there is no native support for this.
